### ISSUE TITLE
[BUG] freeze and crash visual studio

### ISSUE THREAD

**Issue by Cologler** [canton7/PropertyChanged.SourceGenerator#24]
**Description**

Keep crash visual studio if add such code:

**To Reproduce**

1. Create an empty project, target to .Net 6.
2. Paste code, visual studio should freeze, then crash:

``` cs
using PropertyChanged.SourceGenerator;

namespace Test1;
partial record Class1(Class1 Class1)
{
    [Notify] string _value;
}
```

**Version Info**
 - PropertyChanged.SourceGenerator version: 1.0.5
 - Roslyn version: 4.3.0-3.22470.13 (80a8ce8d)
 - Visual Studio version: 17.3.5

**Comment by canton7** [comment]
Good spot, thanks!

**Comment by canton7** [comment]
This will be fixed in 1.0.6 shortly

**Comment by Cologler** [comment]
You are the fastest developer I have met.
